#!/bin/sh /usr/share/dpatch/dpatch-run
## 01-makefiles.dpatch by Fabian Greffrath <fabian@debian-unofficial.org>
##
## DP: Fix upstream makefiles

@DPATCH@

diff -Nru unace-nonfree-2.5.orig/base.mk unace-nonfree-2.5/base.mk
--- unace-nonfree-2.5.orig/base.mk	2003-11-27 20:16:20.000000000 +0100
+++ unace-nonfree-2.5/base.mk	2006-01-31 09:33:04.000000000 +0100
@@ -427,9 +427,7 @@
 ##############################################################################
 
 ifndef ISWMAKE
-include $(MAKE_DIR)baseace.mk
 include $(MAKE_DIR)baseuace.mk
 else
-!include $(MAKE_DIR)baseace.mk
 !include $(MAKE_DIR)baseuace.mk
 endif
diff -Nru unace-nonfree-2.5.orig/makefile unace-nonfree-2.5/makefile
--- unace-nonfree-2.5.orig/makefile	2003-11-24 00:24:24.000000000 +0100
+++ unace-nonfree-2.5/makefile	2006-01-31 09:40:17.000000000 +0100
@@ -18,6 +18,6 @@
 INCLDIR 	= //usr/include
 
-DEFINES		= -D__LINUX__ -D__unix__ -D__GCC__ -D__HAS_IO_H__
+DEFINES		= -D__LINUX__ -D__unix__ -D__GCC__ -U__HAS_IO_H__
 
 ifdef DEBUG
 LSWITCHES	=
@@ -47,7 +47,7 @@
   $(APPS_UNACEEXE_CFILES)
 
 unace: $(UNACEEXELIN_CFILES)
-	gcc $(LSWITCHES) -Wl,-lncurses -static -I$(INCLDIR) -I$(INCLSYSDIR) -I$(SRCSsl) $(DEFINES) $(UNACEEXELIN_CFILES) -ggdb -o$(EXECS_DIR)unace
+	gcc $(LSWITCHES) -Wl,-lncurses -I$(INCLDIR) -idirafter $(INCLSYSDIR) -I$(SRCSsl) $(DEFINES) $(UNACEEXELIN_CFILES) -ggdb -o$(EXECS_DIR)unace
 ifndef DEBUG
 	tar cfvz linunace25.tgz unace file_id.diz licence
 	#sh linpack.sh
diff -Nru unace-nonfree-2.5.orig/unace.mk unace-nonfree-2.5/unace.mk
--- unace-nonfree-2.5.orig/unace.mk	2003-11-27 20:17:46.000000000 +0100
+++ unace-nonfree-2.5/unace.mk	2006-01-31 09:33:21.000000000 +0100
@@ -27,8 +27,6 @@
 
 ifndef ISWMAKE
 include $(MAKE_DIR)unaceexe.mk
-include $(MAKE_DIR)unacedll.mk
 else
 !include $(MAKE_DIR)unaceexe.mk
-!include $(MAKE_DIR)unacedll.mk
 endif
